src/components/Calendar/calendar.css

```diff
@@ -36,6 +44,11 @@
     font-size: 0.9rem;
 }
 
+.rbc-label.dark-mode {
+    font-size: 0.9rem;
```

Comment:
Same idea as the previous comment, why would the font size need to change for dark mode?